sos/report/plugins/krb5.py

```diff
@@ -9,6 +9,7 @@
 # See the LICENSE file in the source distribution for further information.
 
 from sos.report.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+import socket
```

Comment:
Nitpic, but in other plugins, like 'ceph_common', 'pcp', and 'qpid_dispatch' we do this instead:

`from socket import gethostname`

So it may be good to do the same here.

Reply:
Did the suggested changes and move this above the sos.report.plugin import